Camera.MAUI/Camera.MAUI.csproj

```diff
@@ -68,4 +69,17 @@
 	  </MauiXaml>
 	</ItemGroup>
 
+  <!-- Make sure any old temp files are removed-->
+  <Target Name="CustomDroidClean" AfterTargets="Clean" Condition="$([MSBuild]::IsOSPlatform('Windows'))">
+    <Message Text="Running custom clean up bin... $(ProjectDir)bin" />
+    <Exec Command="rmdir /s /q $(ProjectDir)bin" />
+    <Message Text="Running custom clean up obj... $(ProjectDir)obj" />
+    <Exec Command="rmdir /s /q $(ProjectDir)obj" />
+  </Target>
+  <Target Name="CustomDroidCleanMac" AfterTargets="Clean" Condition="$([MSBuild]::IsOSPlatform('OSX'))">
+    <Message Text="Running custom clean up bin... $(ProjectDir)bin" />
+    <Exec Command="rm -R $(ProjectDir)bin" ContinueOnError="true" />
+    <Message Text="Running custom clean up obj... $(ProjectDir)obj" />
+    <Exec Command="rm -R $(ProjectDir)obj" ContinueOnError="true" />
+  </Target>
```

Comment:
Some small comments on this part:
* Nothing in this is Android-specific, so having `Droid` in the name is unnecessary.
* The OSX part should work on Linux as well, and the condition should be extended accordingly. The name could have the suffix `Unix` instead of `Mac` then.

(The same comments apply to the corresponding part in `Camera.MAUI.Test.csproj`.)

Reply:
Ha, yah.  That's a copy-paste re-use I've been using for some time.  The name is due to Android builds being the prime culprit needing this help, when switching between branches.  And... never even tried to dev maui stuff on linux, so never even crossed my mind.  😅

Reply:
It also reclaims quite a bit of disk space.  Android builds are also quite messy on disk.

Reply:
I think it could simply be:
`[MSBuild]::IsOSPlatform('OSX') Or [MSBuild]::IsOSPlatform('Linux')`